doc/diag_comp_overall_architecture.uxf

```diff
@@ -42,347 +67,405 @@ UDP package</panel_attributes>
     <panel_attributes>lt=-&gt;
-Stop 
-request</panel_attributes>
-    <additional_attributes>10.0;10.0;10.0;60.0</additional_attributes>
+StopRequest
+group=1</panel_attributes>
+    <additional_attributes>10.0;10.0;10.0;90.0</additional_attributes>
   </element>
   <element>
     <id>UMLState</id>
     <coordinates>
-      <x>343</x>
-      <y>518</y>
-      <w>77</w>
-      <h>28</h>
+      <x>504</x>
+      <y>642</y>
+      <w>108</w>
+      <h>24</h>
     </coordinates>
-    <panel_attributes>Wait for
-stop reply</panel_attributes>
+    <panel_attributes>halign=center
+valign=center
+WaitForStopReply
+group=1</panel_attributes>
     <additional_attributes/>
   </element>
   <element>
     <id>Relation</id>
     <coordinates>
-      <x>259</x>
-      <y>315</y>
-      <w>224</w>
-      <h>259</h>
-    </coordinates>
-    <panel_attributes>lt=-&gt;
-m1=StopReply received</panel_attributes>
-    <additional_attributes>170.0;330.0;170.0;350.0;10.0;350.0;10.0;10.0;120.0;10.0</additional_attributes>
-  </element>
-  <element>
-    <id>Relation</id>
-    <coordinates>
-      <x>413</x>
-      <y>434</y>
-      <w>98</w>
-      <h>49</h>
+      <x>606</x>
+      <y>570</y>
+      <w>120</w>
+      <h>30</h>
     </coordinates>
     <panel_attributes>lt=-&gt;
-Monitoring 
-frame
-received</panel_attributes>
-    <additional_attributes>10.0;10.0;50.0;10.0;50.0;50.0;10.0;50.0</additional_attributes>
+RawMonitoringFrameReceived
+group=1</panel_attributes>
+    <additional_attributes>10.0;10.0;30.0;10.0;30.0;30.0;10.0;30.0</additional_attributes>
   </element>
   <element>
     <id>UMLGeneric</id>
     <coordinates>
-      <x>224</x>
-      <y>252</y>
-      <w>287</w>
-      <h>329</h>
+      <x>384</x>
+      <y>384</y>
+      <w>378</w>
+      <h>348</h>
     </coordinates>
     <panel_attributes>symbol=component
+style=wordwrap
+layer=0
+group=1
 &lt;&lt;State machine&gt;&gt;
-Scanner Controller
-(possible realization; not fixed)
---</panel_attributes>
-    <additional_attributes/>
-  </element>
-  <element>
-    <id>Relation</id>
-    <coordinates>
-      <x>504</x>
-      <y>406</y>
-      <w>49</w>
-      <h>28</h>
-    </coordinates>
-    <panel_attributes>lt=..
+ScannerProtocolDef
+--
+**Explanation:**
+This component represents the scanner protocol/ state machine (indirectly) described in the Communication manual of the scanner on page 1.
+--
+
+
 </panel_attributes>
-    <additional_attributes>50.0;20.0;10.0;20.0</additional_attributes>
+    <additional_attributes/>
   </element>
   <element>
     <id>UMLGeneric</id>
     <coordinates>
-      <x>539</x>
-      <y>147</y>
-      <w>532</w>
-      <h>133</h>
+      <x>348</x>
+      <y>132</y>
+      <w>456</w>
+      <h>114</h>
     </coordinates>
     <panel_attributes>symbol=component
 style=wordwrap
+layer=-10
 &lt;&lt;User API&gt;&gt;
-Scanner
+ScannerV2
+--
+**Explanation:**
+This abstraction layer represents the user API (ROS-agnostic) and consists of the shown functions.
```

Comment:
"consists of the shown functions." is not necessary. Instead, write what it is doing.